content/bn/_index.md

```diff
@@ -14,10 +14,9 @@ title: "ক্লাউড নেটিভ শব্দকোষ"
 
 ## স্বীকৃতি
 
-ক্লাউড নেটিভ শব্দকোষটি CNCF মার্কেটিং দ্বারা সূচিত হয়েছিল
-কমিটি (CNCF Marketing Committee) এবং অন্তর্ভুক্ত
-থেকে অবদান [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/),
-[Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), [Katelin Ramer](https://www.linkedin.com/in/katelinramer/), [Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca), [Seokho Son](https://www.linkedin.com/in/seokho-son/),[MD Shahriyar Al Mustakim Mitul](https://www.linkedin.com/in/md-shahriyar-al-mustakim-mitul-9084b31a0/), [Umme Abira Azmary](https://www.linkedin.com/in/umme-abira-azmary-68404a1bb/),[MD Ikramul Kayes](https://www.linkedin.com/in/md-ikramul-kayes-753674214/) এবং আরো অনেক অবদানকারী। একটি সম্পূর্ণ অবদানকারী তালিকার জন্য, অনুগ্রহ করে [এই GitHub পৃষ্ঠা](https://github.com/cncf/glossary/graphs/contributors) দেখুন.
+ক্লাউড নেটিভ পরিভাষাটি CNCF মার্কেটিং কমিটি ও বিভিএস দ্বারা প্রবর্তন করা হয়েছিল এবং এতে [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/) , [Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/) , [Katelin Ramer](https://www.linkedin.com/in/katelinramer/) , [Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca), [Seokho Son](https://www.linkedin.com/in/seokho-son/) এবং অন্যান্য অনেক অবদানকারীর অবদান অন্তর্ভুক্ত রয়েছে।
```

Comment:
The English translation of this section with Google Translator is
> **The term Cloud Native was coined** by the CNCF Marketing Committee **and VVS** and includes contributions from Catherine Paganini, Chris Aniszczyk, Daniel Jones, Jason Morgan, Katelin Ramer, Mike Foster, Seokho Son and many others.

which is slightly different from the original English text:

> **The Cloud Native Glossary was initiated** by the CNCF Marketing Committee **(Business Value Subcommittee)** and includes contributions from [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/), [Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), [Katelin Ramer](https://www.linkedin.com/in/katelinramer/), [Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca), and many more contributors. **For a complete contributor list, please refer to [this GitHub page](https://github.com/cncf/glossary/graphs/contributors).**
> 
> **The Glossary is maintained by [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), [Jihoon Seo](https://www.linkedin.com/in/jihoon-seo/), [Noah Ispas](https://www.linkedin.com/in/noah-ispas-0665b42a/), and [Seokho Son](https://www.linkedin.com/in/seokho-son/).**

@mitul3737 Could you check and update one more time? 😊

Reply:
Yes, sure  @jihoon-seo  ☺️

Reply:
Hey @jihoon-seo 
Will this be alright?
`ক্লাউড নেটিভ শব্দকোষটি CNCF মার্কেটিং কমিটি (বিজনেস ভ্যালু সাবকমিটি) দ্বারা সূচিত হয়েছিল এবং এতে [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/) , [Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/) , [Katelin Ramer](https://www.linkedin.com/in/katelinramer/) , [Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca) এবং আরও অনেক অবদানকারীর অবদান অন্তর্ভুক্ত রয়েছে। একটি সম্পূর্ণ অবদানকারী তালিকার জন্য, অনুগ্রহ করে এই [GitHub পৃষ্ঠাটি](https://github.com/cncf/glossary/graphs/contributors) দেখুন।


ক্লাউড নেটিভ ভোকাবুলারির বাংলা স্থানীয়করণের সূচনা [Bengali localization team](https://cloud-native.slack.com/archives/C02UG2WGXQQ) দ্বারা করা হয়েছে এবং এতে অন্তর্ভুক্ত রয়েছে [MD Shahriyar Al Mustakim Mitul](https://www.linkedin.com/in/md-shahriyar-al-mustakim-mitul-9084b31a0/), [MD Ikramul Kayes](https://www.linkedin.com/in/md-ikramul-kayes-753674214/), [Umme Abira Azmary](https://www.linkedin.com/in/umme-abira-azmary-68404a1bb/) এবং আরও অনেক অবদানকারী।`